sdk/python/docs/source/index.rst

```diff
@@ -45,6 +45,12 @@ File Source
     :members:
     :exclude-members: FileOptions
 
+Push Source
+------------------
+
+.. automodule:: feast.data_source.PushSource
```

Comment:
I don't think this is necessary, since there is already
```
Data Source
==================

.. automodule:: feast.data_source
    :members:
    :exclude-members: KafkaOptions, KafkaSource, KinesisOptions, KinesisSource
```
which should include all classes in `feast.data_source`

Reply:
Good catch.